models/public/yolox-tiny/accuracy-check.yml

```diff
@@ -0,0 +1,75 @@
+models:
+  - name: yolox-tiny-onnx
+    launchers:
+      - framework: onnx_runtime
+        model: yolox-tiny.onnx
+        adapter: yolox
+    datasets:
+      - name: ms_coco_detection_80_class_without_background
+
+        preprocessing:
+          - type: bgr_to_rgb
+          - type: resize
+            aspect_ratio_scale: fit_to_window
+            size: 416
+          - type: padding
+            pad_type: right_bottom
+            pad_value: '114, 114, 114'
+          - type: normalization
+            mean: [123.675, 116.28, 103.53]
+            std: [58.395, 57.12, 57.375]
+        postprocessing:
+          - type: resize_prediction_boxes
+            rescale: True
+          - type: nms
+            overlap: 0.65
+          - type: clip_boxes
+            apply_to: prediction
+        metrics:
+          - type: map
+            integral: 11point
+            ignore_difficult: true
+            presenter: print_scalar
+          - name: AP@0.5
+            type: coco_precision
+            max_detections: 100
+            threshold: 0.5
+          - name: AP@0.5:0.05:95
+            type: coco_precision
+            max_detections: 100
+            threshold: '0.5:0.05:0.95'
+
+  - name: yolox-tiny
+    launchers:
+      - framework: dlsdk
+        adapter: yolox
+    datasets:
+      - name: ms_coco_detection_80_class_without_background
+
+        preprocessing:
+          - type: resize
+            aspect_ratio_scale: fit_to_window
+            size: 416
+          - type: padding
+            pad_type: right_bottom
+            pad_value: '114, 114, 114'
```

Comment:
strange number for padding, are you sure that it is correct?

Reply:
This value is taken from the `preproc` function from the original repository https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/data/data_augment.py#L153.